spec/fixtures/authorities/linked_data/lod_full_config_1_0.json

```diff
@@ -0,0 +1,164 @@
+{
+  "prefixes": {
```

Comment:
This provides a single config that is still in 1.0 version style to allow testing of the auto-conversion process to 2.0.